editor/src/components/text-editor/text-editor.spec.browser2.tsx

```diff
@@ -446,7 +448,7 @@ describe('Use the text editor', () => {
               data-testid='first-div'
               data-uid='first-div'
             >
-              Hello this
+              Helllo this
```

Comment:
does this test break on current master?

Reply:
No, but it breaks with my modification, because I don't save when there is no change to the text, that is why I had to type something